File: Set-22-2021/ex001.py
#   Programa para N eleitores votarem em 3 candidatos.

nEleitores = int(input("Qual o número de eleitores? "))

i = 0
n1 = n2 = n3 = 0

while i < nEleitores:
    voto = int(input("Em qual candidato o {}° eleitor vota?\nDigite 1, 2 ou 3: ".format(i + 1)))
    
    if (voto == 1):
        n1 += 1
        i += 1
    elif (voto == 2):
        n2 += 1
        i += 1
    elif (voto == 3):
        n3 += 1
        i += 1
    else:
        print("\nValor inválido, por favor digite novamente.")

print("\nO primeiro candidato conseguiu {} votos".format(n1))
print("O segundo candidato conseguiu {} votos".format(n2))
print("O terceiro candidato conseguiu {} votos".format(n3))

File: Set-22-2021/ex001.c
/*
    Programa para N eleitores votarem em 3 candidatos.
*/
#include <stdio.h>

int main()
{
    int nEleitores, n1=0, n2=0, n3=0, i, voto;
    
    printf("Qual o número total de eleitores? ");
    scanf("%d", &nEleitores);
    
    for (i=0; i<nEleitores; i++){
        printf("Para qual candidato o %d° eleitor vota?\nDigite 1, 2, ou 3: ", i + 1);
        scanf("%d", &voto);
        if (voto == 1){
            n1++;
        }
        else if (voto == 2){
            n2++;
        }
        else if (voto == 3){
            n3++;
        }
        else{
            printf("\nNúmero inválido, digite novamente.\n");
            i--;
        }
    }
    printf("\nO primeiro candidato conseguiu %d votos", n1);
    printf("\nO segundo candidato conseguiu %d votos", n2);
    printf("\nO terceiro candidato conseguiu %d votos", n3);
}
